samples/matter/template/sample.yaml

```diff
@@ -1,29 +1,21 @@
 sample:
-  name: Matter Template
   description: Matter Template sample
+  name: Matter Template
 tests:
-  matter.template.debug:
+  sample.matter.template.debug:
     build_only: true
-    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf21540dk_nrf52840
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
       - nrf21540dk_nrf52840
-    tags: ci_build
-  matter.template.release:
-    build_only: true
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf21540dk_nrf52840
-    integration_platforms:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf21540dk_nrf52840
     tags: ci_build
-    extra_args: CONF_FILE=prj_release.conf
-  matter.template.no_dfu:
```

Comment:
Please rename step for sample.matter.template.no_dfu: and do not remove it.

Reply:
Done